docs/source/tutorial/windows.rst

```diff
@@ -6,21 +6,21 @@ Prerequisites for Windows
 MSYS2
 -----
 
-To build Idris 2 on Windows, an Unix-like environment is needed for
+To build Idris 2 on Windows, a Unix-like environment is needed for
 all the utilities used during the build. `MSYS2 <https://www.msys2.org>`_ provides that for us.
 
-    1. Download the latest version of MSYS2
-    2. Run the installer. Don't install it under Program files
+    1. Download the latest version of MSYS2.
+    2. Run the installer. Don't install it under Program Files,
        as it needs to write files (the "unix" home directory lies
-       under there, for example)
+       under there, for example).
     3. In the directory where you installed MSYS2, find the file
        ``mingw64.ini`` and add the line ``MSYS2_PATH_TYPE=inherit``.
        This adds the normal windows PATH to the shell in MSYS2.
-    4. Start MSYS2 (click on mingw64.exe, as the icon in the start
-       menu won't pick up the MSYS2_PATH_TYPE from the ini, it can be
-       added to the system settings)
+    4. Start MSYS2 (click on mingw64.exe, as the icon in the Start menu
+       won’t pick up the MSYS2_PATH_TYPE from the .ini file — it can
```

Comment:
Unicode single quote. And another em-dash.

Reply:
I think this qualifies as an interruption. Neither comma nor semicolon seem to fit here.

Reply:
In which case I think it should be an en-dash  : )

Reply:
Done.